demo-site/utils/cdn_deploy_tool_utils.js

```diff
@@ -0,0 +1,82 @@
+const fs = require('fs');
+const path = require('path');
+const { execSync } = require('child_process');
+const { loadJson, saveJson } = require("./utils");
+const { 
+    CDN_DEPLOY_TOOL_RUN_CLI_COMMAND,
+    CDN_DEPLOY_TOOL_CONFIG_FILE_NAME,
+    CDN_DEPLOY_TOOL_BUILD_COMMAND
+} = require("./constants");
+
+const setCDNDeployToolPipeline = (deployToolPath, cdnType, deployType) => {
```

Comment:
This file is definitely not necessary - can be removed

Reply:
😆